tools/api-markdown-documenter/package.json

```diff
@@ -62,6 +62,7 @@
     "@microsoft/api-extractor-model": "^7.23.0",
     "@microsoft/tsdoc": "^0.14.1",
     "@rushstack/node-core-library": "^3.50.1",
+    "async": "^3.2.2",
```

Comment:
In this package the issues come from
- `@fluidframework/build-tools`
- `@fluidframework/server-lambdas`
- `@fluidframework/server-lambdas-driver`
- `nconf@0.12.0` as a transitive dependency of `@fluidframework/server-lambdas` and `@fluidframework/server-services-core`... but that's the latest released version of `nconf` so we might need to force `async@3.2.2` here. Ideally the packages that directly depend on it should, but for an internal tool it might be ok to do it here too.